README.md

```diff
@@ -12,3 +13,102 @@ Supported APIs for PettingZoo
 * DMLab Environments
 
 We are open to supporting more external APIs, please create an issue or ideally, a pull request implementing the new API.
+
+## Installation
+
+To install Shimmy from PyPI:
+```
+pip install shimmy
+```
+Out of the box, Shimmy doesn't install any of the dependencies required for the environments it supports.
+To install them, you'll have to install the optional extras.
+The list of extras are as follows:
+
+### OpenAI Gym
+```
+pip install shimmy[gym]
+```
+
+### Atari Environments
+```
+pip install shimmy[atari]
+```
+
+### DM Control (both single and multiagent environments)
+```
+pip install shimmy[dm-control]
+```
+
+### OpenSpiel
+```
+pip install shimmy[pettingzoo]
+```
+
+### For Developers and Testing Only
+```
+pip install shimmy[testing]
+```
+
+### To just install everything
+```
+pip install shimmy[all, testing]
+```
+
+## Usage
+
+All single agent environments have registration under the Gymnasium API, while all multiagent environments must be wrapped using the corresponding compatibility wrappers.
+
+###  OpenAI Gym
+
+Mark to do
```

Comment:
I would check but Gymnasium.make("GymV22Compatibility-v0", env_name"...") and add a env